wpimath/src/main/native/cpp/controller/ProfiledPIDController.cpp

```diff
@@ -9,4 +9,5 @@ void frc::detail::ReportProfiledPIDController() {
   ++instances;
   wpi::math::MathSharedStore::ReportUsage(
       wpi::math::MathUsageId::kController_ProfiledPIDController, instances);
+  wpi::SendableRegistry::Add(this, "ProfiledPIDController", instances);
```

Comment:
this needs to be done in `PIDController.cpp`

Reply:
It's already done there

Reply:
Sorry, meant to say it needs to be done in `ProfiledPIDController.h`- `ReportProfiledPIDController()` isn't a member function of ProfiledPIDController and doesn't have access to `this`